src/PageContent/Calendar/EventCard.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import './EventCard.css';
+import green_truck from '../../icons/green_truck.svg';
+import grey_truck from '../../icons/grey_truck.svg';
+
+class EventCard extends Component {
+    render() {
+        let type = '';
+        if (this.props.eventType === 'recurring') {
```

Comment:
compare to the enum created instead of a string: `this.props.eventType === DeliveryType.RECURRING`